server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/FileIntoAction.java

```diff
@@ -104,11 +97,6 @@ public void execute(ActionFileInto anAction, Mail aMail, final ActionContext con
             }
             throw ex;
         }
-        finally
```

Comment:
We don't want to ghost the mail if only one guy executes his SIEVE script (bug from previous implementation)

Moreover, we define that a mail do not need to be relayed for a given recipient if SIEVE script was executed and no storage action was taken for him. (See later in Sieve mailet)